services/authService.js

```diff
@@ -38,4 +38,19 @@ export function getAllUsers() {
     return users;
 }
 
+export function setUserforEdit(email) {
+    localStorage.setItem('userForEdit', email);
+}
+
+export function getUserforEdit() {
+    const email = localStorage.getItem('userForEdit');
+    const user = getUser(email);
```

Comment:
`return getUser(email)`

you write just this

Reply:
I've corrected this. Thanks.